### DISCUSSION TITLE
Can we have a GTA style dialogue

### DISCUSSION THREAD

**Discussion by tsingyayin** [AlessioGr/NotQuests#226] [Feature Requests]
Whether notquest can use the actionbar position to display messages to obtain the story text style similar to that of GTA5, that is, display text at the bottom of the screen. And can we give multiple lines of storylines a time to gradually appear instead of letting them appear together?

**Comment by AlessioGr** [comment]
@tsingyayin 
> Whether notquest can use the actionbar position to display messages to obtain the story text style similar to that of GTA5, that is, display text at the bottom of the screen

Could you give an example for that - maybe a screenshot?

> And can we give multiple lines of storylines a time to gradually appear instead of letting them appear together?

This has been added in the latest version ^^ You can now add a delay for each conversation line, speaker, conversation or category-wide

**Comment by tsingyayin** [comment]
![image](https://user-images.githubusercontent.com/77230422/200446706-8d82d798-327a-40a7-98e9-fd2385ce4c08.png)

I didn't use English for the game. The picture above is in traditional Chinese. In a word, the location of the story text is at the bottom of the game. However, since there is something in Minecraft, I think it may be an ActionBar.

By the way, I feel that you haven't updated the tutorials and documents on the wiki for a long time. I'm sorry that I may not understand some new features.

Thank you for your NotQuests project and your contribution to the open source community.
